install.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
+home_dir="gnu-koryavov"
 read -p "Вы уверены, что хотите установить скрипт gnu-koryavov в систему? (Д/н): " ans
 
 if [[ $ans == "y"* || $ans == "Y"* || $ans == "д"* || $ans == "Д"* ]]; then
 
-    mkdir -p -v ~/gnu-koryavov/KORYAVNIKS
-    cp config.conf ~/gnu-koryavov
-    sudo cp run.sh /usr/local/bin/gnu-koryavov
+    mkdir -p -v ~/$home_dir/KORYAVNIKS
+    cp config.conf ~/$home_dir
+    sudo cp run.sh /usr/local/bin/$home_dir
```

Comment:
Maybe it would be better to keep script name constant? Idk what's the best way to do so, because there are 2 ways available.
`run.sh` filename can be changed or another script name constant can be provided for this part of `install.sh`.

Reply:
I mean

```shell
sudo cp run.sh /usr/local/bin/$scriptname
```

But I'm not sure the script name should be changed. I mean I like `gnu-koryavov` ))